bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/internal/values/TupleValueImpl.java

```diff
@@ -828,4 +824,24 @@ private void resetSize(int index) {
             size = index + 1;
         }
     }
+
+    private void validateTupleSizeAndInherentType() {
+
```

Comment:
This new line not needed

Reply:
Fixed with 8bccb83b2dfffeac9e308dd98c12a65af4b1ab72